frontend/src/components/Shared/VideoPlayer/VideoPlayer.jsx

```
@@ -0,0 +1,229 @@
+          >
+            Cancel
+          </button>
+        </div>
+      </div>
+    </div>
+  );
+  
+  // Render loading indicator
+  const renderLoading = () => (
+    <div className="absolute inset-0 flex items-center justify-center bg-black z-10">
+      <div className="w-12 h-12 border-4 border-blue-500 border-t-transparent rounded-full animate-spin"></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)